payment_redsys/models/redsys.py

```diff
@@ -207,6 +219,12 @@ def _product_description(self, order_ref):
 class TxRedsys(models.Model):
     _inherit = 'payment.transaction'
 
+    # Redsys status
+    _redsys_valid_tx_status = list(range(0, 100))
+    _redsys_pending_tx_status = list(range(101, 203))
+    _redsys_cancel_tx_status = [912, 9912]
+    _redsys_error_tx_status = list(range(9064, 9095))
```

Comment:
Como estás usando estas variables para comparar con el operador `in`, mejor usa `set` en vez de `list`. En cualquier caso, esta implementación penaliza el rendimiento conforme a la anterior.

Creo que lo mejor sería un método parecido a:

```
def _redsys_status(self, code):
    if 0 <= code <= 100:
        return "done"
    if 101 <= code <= 203:
        return "pending"
    ... etc
```